x-pack/plugins/security_solution/public/timelines/components/side_panel/new_user_detail/managed_user.tsx

```diff
@@ -71,6 +75,9 @@ export const ManagedUser = ({
               title={i18n.MANAGED_USER_INSPECT_TITLE}
             />
           }
+          css={css`
+            width: 100%;
+          `}
```

Comment:
I was told yesterday that we should avoiding doing this this way as it will cause rerenders. Instead we should extract `css`width: 100%;` to a constant outside of the component and use it here.
Full disclosure, I tried doing this to some EUI components and failed miserably 😆